docs/source/index.rst

```diff
@@ -23,12 +23,14 @@
 
     X, y = linear_training_data()
     with pm.Model() as linear_model:
-        weights = pm.Normal('weights', mu=0, sigma=1)
-        noise = pm.Gamma('noise', alpha=2, beta=1)
-        y_observed = pm.Normal('y_observed',
-                    mu=X @ weights,
-                    sigma=noise,
-                    observed=y)
+        weights = pm.Normal("weights", mu=0, sigma=1)
+        noise = pm.Gamma("noise", alpha=2, beta=1)
+        y_observed = pm.Normal(
+            "y_observed",
+            mu=X @ weights,
+            sigma=noise,
+            observed=y,
+        )
```

Comment:
this doesn't actually throw a warning or anything, the indentation just looks a bit strange to me